### ISSUE TITLE
va-select - Confirm that we need `aria-live-region-text` in the Storybook examples

### ISSUE THREAD

**Issue by jamigibbs** [department-of-veterans-affairs/vets-design-system-documentation#1483]
## Description
The `va-select` Storybook examples have an `aria-live-region-text` attribute and we're unsure if it's necessary:

https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/storybook/stories/va-select.stories.jsx#L25

I wasn't able to find any documentation about when or why this was added to Storybook. This doesn't seem to be an attribute of the component itself. It's just added to `va-select.stories.jsx`.

![Screenshot 2023-01-30 at 12 48 10 PM](https://user-images.githubusercontent.com/872479/215567709-1f48c93e-0699-43f1-a579-4634b9b8c243.png)

Based on this [Slack conversation with Josh Kim](https://dsva.slack.com/archives/C8E985R32/p1675095738560739), it could be redundant (e.g. “You selected x, selected”).

## Tasks
- [ ] Confirm if the attribute `aria-live-region-text="You selected"` is necessary in our Storybook documentation for va-select.
- [ ] If the attribute is not necessary, remove it from the Storybook examples.

## Acceptance Criteria
- [ ] The existence of `aria-live-region-text` is validated and Storybook updated (if necessary).
